### ISSUE TITLE
How to disable sandbox mode

### ISSUE THREAD

**Issue by itpropro** [soarqin/DSP_Mods#3]
When enabling sandbox mode with the mod, adding a few veins and disabling it again, how can you avoid that the save is flagged with sandbox mode and how do you remove the sandbox mode flag from an existing save.

**Comment by soarqin** [comment]
I don't think you can do this with embedded cheat functions, you may need other mods.
Anyways I may add this as an individual function to CheatEnabler some time.

**Comment by soarqin** [comment]
disable sandbox mode is implemented in official Dev Shortcuts, check in-game tips for details
